include/aikido/planner/vectorfield/VectorFieldPlanner.hpp

```diff
@@ -97,6 +101,7 @@ aikido::trajectory::UniqueInterpolatedPtr planToEndEffectorOffset(
 /// \return Trajectory or \c nullptr if planning failed.
 aikido::trajectory::UniqueInterpolatedPtr planToEndEffectorPose(
     const aikido::statespace::dart::MetaSkeletonStateSpacePtr& stateSpace,
+    const statespace::dart::MetaSkeletonStateSpace::State& startState,
```

Comment:
Add docstring for this parameter below line 85.

Reply:
Actually, why does this method need to change in this PR? Please make a separate PR for this and have this PR be only about `planWithEndEffectorTwist`